components/frontend/src/subject/Subject.js

```diff
@@ -107,7 +107,8 @@ export function Subject({
 
     useEffect(() => {
         if (dates.length > 1) {
-            get_subject_measurements(subject_uuid, report_date).then(json => {
+            const minReportDate = dates.slice().sort((d1, d2) => {return (d1.toISOString() > d2.toISOString())}).at(0);
```

Comment:
wow magical notation

Reply:
Yeah, crappy JS. slice() is needed to make a copy because sort() alters the array. It seems to be the "canonical" way to do this.